src/llmcompressor/pipelines/cache.py

```diff
@@ -31,28 +34,17 @@ class IntermediatesCache:
     Construct using `empty` and `from_dataloader` class methods
     """
 
-    batch_intermediates: List[Dict[str, IntermediateValue]]
+    batch_intermediates: List[IntermediateValues]
     offload_device: torch.device
 
     def __init__(
         self,
-        batch_intermediates: List[Dict[str, IntermediateValue]],
-        offload_device: torch.device,
+        batch_intermediates: Optional[List[IntermediateValues]] = None,
+        offload_device: torch.device = "cpu",
     ):
-        self.batch_intermediates = batch_intermediates
+        self.batch_intermediates = batch_intermediates or []
         self.offload_device = offload_device
 
-    @classmethod
-    def empty(cls, num_batches: int, offload_device: torch.device):
```

Comment:
@kylesayrs `IntermediatesCache.empty` is used in `src/llmcompressor/pipelines/layer_sequential/helpers.py`, should we retain this?

Reply:
Yes, my mistake for deleting it